modules/govuk_unattended_reboot/manifests/init.pp

```diff
@@ -59,6 +66,26 @@
     require => File['/usr/local/bin/check_icinga'],
   }
 
+  $lock_dir = "${config_directory}/no-reboot"
+
+  file { $lock_dir:
+    ensure  => directory,
+    mode    => '0777',
+    owner   => 'root',
+    group   => 'root',
+    require => File[$config_directory],
+  }
+
+  file { 'no_reboot':
+    ensure  => file,
+    path    => "${check_scripts_directory}/no_reboot",
```

Comment:
Should this be named with a numbered prefix to match [what we already do](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_unattended_reboot/manifests/init.pp#L50)?